source/usage-examples/findOne.txt

```diff
@@ -3,3 +3,95 @@ Find a Document
 ===============
 
 .. default-domain:: mongodb
+
+You can retrieve a single document from your collection with the
+``*Collection.FindOne()`` method. 
+
+..
+   method
+
+The ``FindOne()`` method takes a query document that specifies the
+subset of your collection that you would like to retrieve a document from. If you
```

Comment:
The description of of the query document specifying a subset seems a little off. Maybe something like "specifies a filter that will be applied to the collection. Only documents that match the filter will be returned by the server. For example, if the filter document is {x: 1}, the document {x: 1, y: 1} would match but {x: 2, y: 1} would not".

Reply:
Thanks for the note. After implementing https://github.com/mongodb/docs-golang/pull/2#discussion_r617956487 the edit "specifies a filter" creates some circular circular language (takes a filter document that specifies a filter). I have tried to rewrite the section inline with your edit, but changing it to fit within your other suggestions. Please let me know if this captures your meaning: https://github.com/mongodb/docs-golang/pull/2/commits/91b48490f95a3a520f1aacf8e0c3ee957e982f8b

While I agree that specifying sample documents inline may be helpful for some readers, I believe our usage example documentation generally favors directing readers to the end of page example over providing examples in the introductory section.

Reply:
Apologies for not following up in this thread sooner. I have restarted the thread on the DevEd team asking about inline sample documents in usage example prose (thread did not reach resolution). Will try and follow up here with the results of that conversation.